docs/tutorials/Monge_Gap.ipynb

```diff
@@ -1,5 +1,15 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    from ott.neural.solvers import losses, map_estimator
</pre><p>This needs to be adjusted as </p><p>from ott.neural import losses</p><p>from ott.neural.solvers import map_estimator</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/467/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>